### ISSUE TITLE
Executable name on Forge builds doesn't take account of packagerConfig

### ISSUE THREAD

**Issue by reubenscratton** [webdriverio-community/wdio-electron-service#621]
application.ts line 52 is:

```
        darwin: path.join(`${appBuildInfo.appName}.app`, 'Contents', 'MacOS', appBuildInfo.appName),
```

I believe this should be:

```
        darwin: path.join(`${appBuildInfo.appName}.app`, 'Contents', 'MacOS', appBuildInfo.config.packagerConfig.executableName),
```
This change gets me further towards being able to use WebdriverIO with my ElectronJS project, although there is still a way to go. I think there are assumptions about project structure that aren't quite true, e.g. you use EITHER Forge or Builder but never both. But my project uses both - Forge for MacOS and Builder for Windows because of various bugs.

**Comment by goosewobbler** [comment]
@reubenscratton please raise a separate issue for the multiple build tool approach with whatever detail you have around your use case, so that it can be discussed there.

**Comment by goosewobbler** [comment]
@reubenscratton This is released in `7.0.0-beta.1`.  You will need to use the WDIO v9 pre-release with it.
